build.zig

```diff
@@ -2,9 +2,14 @@ const std = @import("std");
 const builtin = @import("builtin");
 const CrossTarget = std.zig.CrossTarget;
 const Mode = std.builtin.Mode;
+const file_or_directory_exists = @import("src/clients/shutil.zig").file_or_directory_exists;
 
 const config = @import("./src/config.zig");
 
+// We need to ensure we only add a clone dependency once.
+var git_clone_tracy = false;
+var git_clone_jui = false;
```

Comment:
:thinking: this is not quite right I think. Zig build system is a two-phase system:

* there's "configure" phase which means running `pub fn build` and constructing a graph of steps
* then there's "execution" phase, where a subset of the step graph is run in the dependency order

I think what happens here is mixing the two phases: you check the flag in the configure phase, but you actually need to check if the dir exists at the execution phase. 

I think that'd be easy to do with https://github.com/tigerbeetledb/tigerbeetle/pull/950, so I suggest maybe merging as is, and I clean this right after that?

Reply:
> I think what happens here is mixing the two phases: you check the flag in the configure phase, but you actually need to check if the dir exists at the execution phase.

Yes, it's exactly that - this is just a workaround :) Fixing it with #950 after sounds good to me!